essentials/teleport.dsc

```diff
@@ -0,0 +1,210 @@
+      - else if <context.args.first.is_integer> || <context.args.first> == ~:
+        - determine ~
+
+    - else if <[arg_count]> == 3:
+      - if !<context.args.remove[first].exclude[~|~|~].parse[is_integer].contains[false]> || <server.match_player[<context.args.first>].if_null[invalid]> == invalid:
+        - determine ~
+
+    - else if <[arg_count]> == 4:
+      - if <server.match_player[<context.args.first>].if_null[invalid]> == invalid:
+        - if !<context.args.get[1].to[3].exclude[~|~|~].parse[is_integer].contains[false]>:
+          - determine <server.worlds.parse[name].filter[starts_with[<context.args.get[4].if_null[<empty>]>]]>
+      - else if !<context.args.remove[first].exclude[~|~].parse[is_integer].contains[false]>:
+        - determine ~
+
+    - else if <[arg_count]> == 5 && <server.match_player[<context.args.first>].if_null[invalid]> != invalid && !<context.args.get[2].to[4].exclude[~|~|~].parse[is_integer].contains[false]>:
+      - determine <server.worlds.parse[name].filter[starts_with[<context.args.get[5].if_null[<empty>]>]]>
+
+  help:
+    - narrate "<&6>/<&e>teleport x y z <&b>| <&a>Teleports you to the coordinates <&2><&lt><&a>x y z<&2><&gt>"
+    - narrate "<&6>/<&e>teleport x y z world <&b>| <&a>Teleports you to <&2><&lt><&a>x y z<&2><&gt><&a> on the named world"
+    - narrate "<&6>/<&e>teleport world <&b>| <&a>Teleports you to the named world"
+    - narrate "<&6>/<&e>teleport PlayerTwo <&b>| <&a>Teleports you to PlayerTwo"
+    - narrate "<&6>/<&e>teleport PlayerOne x y z <&b>| <&a>Teleports PlayerOne to <&2><&lt><&a>x y z<&2><&gt>"
+    - narrate "<&6>/<&e>teleport PlayerOne x y z world <&b>| <&a>Teleports PlayerOne to <&2><&lt><&a>x y z<&2><&gt><&a> on the named world"
+    - narrate "<&6>/<&e>teleport PlayerOne world <&b>| <&a>Teleports PlayerOne to the named world"
+    - narrate "<&6>/<&e>teleport PlayerOne PlayerTwo <&b>| <&a>Teleports PlayerOne to PlayerTwo"
+
+  check:
+    coordinates:
+      - foreach <[coordinates]> as:coordinate:
+        - if !<[coordinate].is_integer>:
+          - if <[coordinate]> == ~:
+            - define index <player.location.simple.before_last[,].split[,].get[<[loop_index]>]>
+            - define coordinates <[coordinates].set[<[index]>].at[<[loop_index]>]>
+            - foreach next
+          - define invalid_coordinates:|:<[coordinate]>
+      - if <[invalid_coordinates].exists>:
+        - if <[invalid_coordinates].size> > 1:
+          - define grammar_verb are
+          - define grammar_noun coordinates
+        - else:
+          - define grammar_verb "is an"
+          - define grammar_noun coordinate
+        - define reason "<[invalid_coordinates].formatted> <[grammar_verb]> invalid <[grammar_noun]>."
+        - inject command_error
+
+      - foreach <[coordinates]> as:coordinate:
+        - if <[coordinate].abs> > 9000:
```

Comment:
should this be 8000?

Reply:
I'll be honest and confess i both:
- intentionally lied to my players here to ensure they didn't get too close to the world border i hard-coded
- am unsure of what the real world border is lol

Reply:
take advantage of `WorldTag.border_size`

Reply:
This tag would be invalid as the server doesn't actually use the world border; it has some form of event handling for when a player exceeds the allowed limits

we do how-ever need clarification on the world border